src/manifest.json.tpl

```diff
@@ -8,6 +8,9 @@
     "96": "icons/lb_locked.svg"
   },
 
+
+  "content_security_policy": "script-src 'self' {{csp_scripts}} ; object-src 'self' {{csp_objects}}",
```

Comment:
Since this is a template file, not just straight json, maybe add a comment to explain that the CSP rule is needed to allow the a11y test to inject code into a running page

Reply:
While a template file, the template language isn't that sophisticated, but I'll see what I can do to approximate a comment.

Reply:
Feel free to ignore this suggestion if it makes things too complicated; future people can always rely on git blame to find their way to some context :-)

Reply:
it's kind of turning into that \-:

This project is using [json-templater](https://github.com/lightsofapollo/json-templater) to turn it into a full-fledged JSON file.  It supports a simplified mustache syntax via a hand-rolled parser.

I could replace this module with something else, but this was the best match we found the first and second times through.  The alternative seems to be to roll own own, which I'm really not inclined to do ...

Reply:
changes the `csp_` variable names to note they're for testing purposes